09-hypothesis-testing.Rmd

```diff
@@ -1338,28 +1346,30 @@ all_points <- tibble(
   `t: df = 3` = dt(x = domain, df = 3),
   `t: df = 10` = dt(x = domain, df = 10),
   `z` = dnorm(x = domain)
-)  %>% 
-  gather(key = "Distribution", value = "value", - domain) %>% 
+) %>%
+  gather(key = "Distribution", value = "value", -domain) %>%
   mutate(
     type = ifelse(Distribution == "z", "z", "t"),
     Distribution = factor(
-      Distribution, 
+      Distribution,
       levels = c("z", "t: df = 10", "t: df = 3", "t: df = 1")
-      )
     )
+  )
 
-for_labels <- all_points %>% 
+for_labels <- all_points %>%
   filter(between(domain, -1.205, -1.195) & Distribution == "t: df = 1" |
-         between(domain, -0.205, -0.195) & Distribution == "t: df = 3" |
-         between(domain, 0.095, 0.105) & Distribution == "t: df = 10" |
-         domain == 2.2 & Distribution == "z")
+    between(domain, -0.205, -0.195) & Distribution == "t: df = 3" |
+    between(domain, 0.095, 0.105) & Distribution == "t: df = 10" |
+    domain == 2.2 & Distribution == "z")
 
-all_points %>% 
+all_points %>%
   ggplot(aes(x = domain, y = value, color = Distribution)) +
   geom_line(aes(linetype = type)) +
   scale_color_viridis(discrete = TRUE, begin = 0, end = 0.7) +
-  ggrepel::geom_label_repel(data = for_labels, aes(label = Distribution),
-                            nudge_x = c(1, -2, 2, -2)) +
+  ggrepel::geom_label_repel(
+    data = for_labels, aes(label = Distribution),
```

Comment:
For some reason to me aesthetically (no pun intended), I like having each argument appear on its own line so something like

```
ggrepel::geom_label_repel(
    data = for_labels,
    mapping = aes(label = Distribution),
    nudge_x = c(1, -2, 2, -2)
)
```
is a lot easier to read for me. What do you think?

Reply:
huh, I thought `styler` autofixed these. sounds good

Reply:
also now we now load `library(ggrepel)` in the "internal packages needed" section on top